core/storage/collection/gae_models.py

```diff
@@ -237,10 +246,20 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
-    @staticmethod
-    def get_export_policy():
+    @classmethod
+    def get_export_policy(cls):
         """Model contains user data."""
-        return base_models.EXPORT_POLICY.CONTAINS_USER_DATA
+        return dict(super(cls, cls).get_export_policy(), **{
+            'owner_ids': base_models.EXPORT_POLICY.EXPORTED,
+            'editor_ids': base_models.EXPORT_POLICY.EXPORTED,
+            'voice_artist_ids': base_models.EXPORT_POLICY.EXPORTED,
+            'viewer_ids': base_models.EXPORT_POLICY.EXPORTED,
+            'community_owned': base_models.EXPORT_POLICY.NOT_EXPORTED,
+            'viewable_if_private': base_models.EXPORT_POLICY.NOT_EXPORTED,
+            'status': base_models.EXPORT_POLICY.NOT_EXPORTED,
+            'first_published_msec': base_models.EXPORT_POLICY.NOT_EXPORTED,
+            'translator_ids': base_models.EXPORT_POLICY.NOT_EXPORTED
```

Comment:
Maybe add a comment above this line saying that the field is deprecated? Otherwise it looks user-related.

Reply:
Added